birdhouse/components/README.rst

```diff
@@ -821,6 +821,42 @@ Usage
 The service is available at ``${BIRDHOUSE_PROXY_SCHEME}://${BIRDHOUSE_FQDN_PUBLIC}/jupyter``. Users are able to log in to Jupyterhub using the
 same user name and password as Magpie. They will then be able to launch a personal jupyterlab server.
 
+GPU support for jupyterlab containers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the host machine has GPUs and you want to make them available to the docker containers running Jupyterlab:
+
+1. ensure that the GPU drivers on the host machine are up to date
+2. install the `NVIDIA container toolkit`_ package on the host machine
+3. configure docker to use the NVIDIA as a container run-time (See the "configure docker" section in `NVIDIA container toolkit`_)
```

Comment:
Better to link directly (https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html#configuring-docker), assuming this is the one you meant?